index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Survey form</title>
+</head>
+
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+
+    <form action="" id="survey-form">
+
+        <label for="name" id="name-label">Name</label>
+        <input type="text" id="name" name="name" placeholder="Enter Your Name" required></input>
+
+        <label for="email" id="email-label">Email</label>
+        <input type="email" id="email" name="email" placeholder="Enter your Email" required></input>
+
+        <label for="number" id="number-label">Age <sup>(Optional)</sup></label>
+        <input type="number" id="number" name="number" min="5" max="35" placeholder="Age"></input>
+
+        <label for="dropdown">Which option best describes your current role?</label>
+        <select name="" id="dropdown">
```

Comment:
- [ ]  I think it is recommended and better to not ❎  leave empty attributes on an element. This is because the browser will still go ahead and attempt to read the empty attributes, causing unnecessary delays. 😀